File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1047.cpp
#include <stdio.h>

int main(){
    int horaI, horaF, minutoI, minutoF, hora, minuto;
    
    scanf("%i %i %i %i",&horaI, &minutoI, &horaF, &minutoF);
    
    minuto = minutoF - minutoI;
    
    hora = horaF - horaI;
    
    if (hora < 0) {
        hora = 24 + (horaF - horaI);
    }
    if (minutoF == minutoI) {
        hora++;
    } else if (minuto < 0) {
        minuto += 60;
        hora -= 1;
    }
    
    if (horaF == horaI) {
        printf("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)\n");
    }else{
        printf("O JOGO DUROU %i HORA(S) E %i MINUTO(S)\n", hora, minuto);
    }
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python/Iniciante/1047.py
# -*- coding: utf-8 -*-

linha = raw_input().split(' ')

horaInicio, minutoInicio, horaFim, minutoFim = linha
horaInicio = int(horaInicio)
horaFim = int(horaFim)
minutoInicio = int(minutoInicio)
minutoFim = int(minutoFim)

hora = horaFim - horaInicio
minuto = minutoFim - minutoInicio

if hora <= 0: #condições
	hora += 24
if minuto < 0:
	hora -= 1
	minuto += 60

print("O JOGO DUROU %i HORA(S) E %i MINUTO(S)" % (hora, minuto))